debian/control

```diff
@@ -37,7 +37,7 @@ Build-Depends: automake,
                libcurl4-openssl-dev,
 # Jaeger       libevent-dev,
                libexpat1-dev,
-# Make-Check   libffi-dev [!amd64],
+               libffi-dev [!amd64] <pkg.ceph.check>,
```

Comment:
might want to drop the line of 

> (cherry picked from commit https://github.com/ceph/ceph/commit/acf9a0b8e95bd8f5412dc0c11f4a5bfc80199104)

in commit message.

Reply:
Fixed, thanks.